### PR TITLE
feat(frontend): dApps carousel styling adjustments

### PR DESCRIPTION
**Pull request by DenysKarmazynDFINITY**
# Motivation

The final approved design:

https://github.com/user-attachments/assets/cb25e5b5-3ef6-47e6-9a24-75b38226d7bd

Approved by Artem ✅

### CODE REVIEW

**Review by peterpeterparker — APPROVED** [review]
LGTm, thx

**Comment by DenysKarmazynDFINITY** [comment]
@peterpeterparker after syncing with Artem, we agreed on polishing a few more things. I pushed them all to this PR, so it has changed a lot since you approved it. Let's re-review it.

**Review by peterpeterparker — APPROVED** [review]
LGTM, thx

File: src/frontend/src/lib/components/dapps/DappsCarousel.svelte [hunk 1]
```diff
@@ -15,7 +15,8 @@
 </script>
 
 {#if nonNullish(dappsCarouselSlides)}
-	<Carousel styleClass={`w-full ${styleClass}`}>
+	<!-- To align controls section with slide text - 100% - logo width (4rem) - margin logo-text (1rem) -->
```
[the remark this answers is not in the document]
Comment (DenysKarmazynDFINITY) [reply src/frontend/src/lib/components/dapps/DappsCarousel.svelte:18]: Definitely, a very nice use-case for a screenshot. I think the screenshot test for logged-in page will cover it - if something breaks style-wise, we'll know 🙂

**Pull request merged**